seedlet-blocks/style.css

```diff
@@ -28,28 +28,42 @@ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 GNU General Public License for more details.
 */
 
-/* Override Seedlet variables with values from experimental-theme.json.
-   Doubling up on the :root selector is weird, but it just ensures these have higher specificity than everything else) */
-:root:root {
-	/* Font Size */
-	--global--font-size-xs: calc( 1px * var(--wp--preset--font-size--tiny) );
-	--global--font-size-sm: calc( 1px * var(--wp--preset--font-size--small) );
-	--global--font-size-md: calc( 1px * var(--wp--preset--font-size--normal) );
-	--global--font-size-lg: calc( 1px * var(--wp--preset--font-size--large) );
-	--global--font-size-xl: calc( 1px * var(--wp--preset--font-size--huge) );
-
-	/* Colors */
-	--global--color-background: var(--wp--preset--color--background);
-	--global--color-foreground: var(--wp--preset--color--foreground);
-	--global--color-primary: var(--wp--preset--color--primary);
-	--global--color-secondary: var(--wp--preset--color--secondary);
-	--global--color-tertiary: var(--wp--preset--color--tertiary);
-}
-
-/* Make sure full-width items in entry content show up as full-width. */
-.wp-block-group.site-content .wp-block-group__inner-container .entry-content {
-	max-width: calc(100% + (2 * var(--global--spacing-horizontal)));
-	width: calc(100% + (2 * var(--global--spacing-horizontal)));
-	margin-left: calc(-1 * var(--global--spacing-horizontal));
-	margin-right: auto;
-}
\ No newline at end of file
+/**
+ * Author bio
```

Comment:
This should go into a sass file

Reply:
https://github.com/Automattic/themes/pull/3575